src/Icicle/Source/ToCore/ToCore.hs

```diff
@@ -80,8 +80,12 @@ convertQueryTop feats qt
                   Just t' -> return t'
         let inpTy'dated = T.PairT inpTy T.DateTimeT
 
-        (bs,ret) <- evalStateT (convertQuery $ query qt) (ConvertState inp inpTy'dated fs Map.empty)
-        let bs'   = strm inp C.Source <> bs
+        (now,bs,ret) <- evalStateT (do
```

Comment:
last night I was talking about taylor swift and laurel said "I've never heard a taylor swift song" and I said "neither have I". no longer can I make that claim, and I hold grudges for a long time